ert/data/record/_transmitter.py

```diff
@@ -115,6 +123,17 @@ async def transmit_file(
                 blob_record = BlobRecord(data=contents_b)
             uri = await self._transmit_blob_record(blob_record)
             self._set_transmitted_state(uri, blob_record.record_type)
+        elif mime == "application/eclsum":
```

Comment:
The serialization package should be able to accommodate ERT in this case as well. The mime should perhaps in this case return an entirely different class of serializer that can produce something a `NumericalRecordStore` can consume?

Or maybe something smarter.

Reply:
I think the borders between `serializer` and `transformation` are a bit blurry and perhaps we should unify these two concepts.

Reply:
Links further to the following issue https://github.com/equinor/ert/issues/2565